wso2

```diff
@@ -0,0 +1 @@
+/Users/viraj/go/src/github.com/VirajSalaka/data-plane-api/wso2
```

Comment:
What is this file

Reply:
symlink created by mistake
fixed via https://github.com/wso2/product-microgateway/pull/1582/commits/14ae7b18ab1203289ee04a49cbda4d7aa5bfdfa6